src/config/interfaces.ts

```diff
@@ -21,11 +13,42 @@ interface ConfigOptions {
   tree?: string;
 }
 
+type ConfigurablePaths =
```

Comment:
Would this work better?
``` Typescript
interface CustomPathConfig {
  authenticate?: string;
  authorize?: string;
  accessToken?: string;
  endSession?: string;
  userInfo?: string;
  revoke?: string;
  sessions?: string;
}

type ConfigurablePaths = keyof CustomPathConfig;

const foo: ConfigurablePaths = 'accessToken' ✅ 
```

Reply:
Very nice! You just taught me something about TypeScript I didn't know about. Thanks!